js/tutorials/photo-albums-in-react/.gitattributes

```diff
@@ -0,0 +1 @@
+*.jekyll linguist-generated
```

Comment:
This _one weird trick_ collapses computed files in PRs, so they're less noisy. 💅